src/main/java/uk/gov/hmcts/darts/common/entity/EventHandlerEntity.java

```diff
@@ -9,13 +9,13 @@
 import jakarta.persistence.Table;
 import lombok.Getter;
 import lombok.Setter;
-import uk.gov.hmcts.darts.common.entity.base.CreatedModifiedBaseEntity;
+import uk.gov.hmcts.darts.common.entity.base.CreatedBaseEntity;
 
 @Entity
 @Table(name = "event_handler")
 @Getter
 @Setter
-public class EventHandlerEntity extends CreatedModifiedBaseEntity {
+public class EventHandlerEntity extends CreatedBaseEntity {
```

Comment:
We seem to be missing the new v52 field - see also [standing data](https://github.com/hmcts/darts-api/blob/master/src/main/resources/db/reference/modernised_darts_standing_data.sql#L21)
`,is_reporting_restriction    BOOLEAN                       NOT NULL`

Reply:
ACCEPT